src/interactive-auth.ts

```diff
@@ -262,6 +263,8 @@ export class InteractiveAuth<T> {
     private readonly requestEmailTokenCallback: IOpts<T>["requestEmailToken"];
     private readonly supportedStages?: Set<string>;
 
+    // The current latest data as part of the interactive auth
+    // MatrixError can occur if the error from server is not a 401 UIA error
     private data: IAuthData | MatrixError;
```

Comment:
I might be being stupid, but can you point me to where a `MatrixError` is assigned to `data` ?

Reply:
Yeah honestly wasn't sure either, was inferring from private consumers. Getting rid of it seems to not break anything 🤷.